README.md

```diff
@@ -16,6 +16,8 @@ Contents
 The [UniMath subdirectory](UniMath/) contains various packages of formalized
 mathematics. For more information about its contents, see the `README` (or `README.md`) in each package.
 
+To get started, see [CONTENTS](CONTENTS.md).
```

Comment:
"For more information about its contents, see the README in each package. To get started, see CONTENTS."

This reads a bit strange: "For the contents, see the README. To get started, see CONTENTS."

Reply:
Yes, the original idea was that each package would display its own contents, as desired.